zeapp/app/src/main/java/de/berlindroid/zeapp/ui/NameEditorDialog.kt

```diff
@@ -33,6 +35,9 @@ import de.berlindroid.zeapp.vm.BadgeViewModel.Configuration
  * @param dismissed callback called when dialog is dismissed / cancelled
  * @param accepted callback called with the new configuration configured.
  */
+
+const val MaxCharacters: Int = 20
```

Comment:
I suggest to yell here 😱 

```suggestion
const val MAX_CHARACTERS: Int = 20
```

Reply:
Compose way ftw folks! :P